app/src/main/kotlin/com/wire/android/util/FileManager.kt

```diff
@@ -0,0 +1,19 @@
+package com.wire.android.util
+
+import android.content.Context
+
+interface FileManager {
+    fun saveToExternalStorage(assetName: String?, assetData: ByteArray, onFileSaved: (String?) -> Unit)
+    fun openWithExternalApp(assetName: String?, assetData: ByteArray, onError: () -> Unit)
+}
+
+class FileManagerImpl(private val context: Context) : FileManager {
+    override fun saveToExternalStorage(assetName: String?, assetData: ByteArray, onFileSaved: (String?) -> Unit) {
+        saveFileToDownloadsFolder(assetName, assetData, context)
+        onFileSaved(assetName)
+    }
+
+    override fun openWithExternalApp(assetName: String?, assetData: ByteArray, onError: () -> Unit) {
+        openAssetFileWithExternalApp(assetName, assetData, context, onError)
+    }
+}
```

Comment:
In `Kalium` we have this limitation of only being able to mock interfaces, as we use Mockative.

But on Reloaded, we have Mockk which allows mocking even of closed classes.
I think it's unnecessary creating the `interface + Impl`. We can just have the `FileManager`.

If we ever have two implementations, then we can use the IDE to help with refactoring and creating an interface for us.